pkg/controller.v1/tensorflow/pod.go

```diff
@@ -70,7 +70,23 @@ func (tc *TFController) reconcilePods(
 
 	initializeTFReplicaStatuses(tfjob, rtype)
 
-	podSlices := tc.GetPodSlices(pods, replicas, logger)
+	podSlices, podsToBeRemoved := tc.GetPodSlices(pods, replicas, logger)
+
+	// Scale down
+	if tfjob.Spec.EnableDynamicWorker && len(podsToBeRemoved) > 0 {
+		// Currently only allow to scale down workers
```

Comment:
will you open another PR for scale up?
or scale up is not in the scope of the proposal?

Reply:
The scaling up has already been taken care by the existing code https://github.com/kubeflow/tf-operator/blob/master/pkg/controller.v1/tensorflow/pod.go#L79